src/main/java/com/ghusse/ci/violations/bbcloud/lib/DiffParser.java

```diff
@@ -0,0 +1,42 @@
+package com.ghusse.ci.violations.bbcloud.lib;
+
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Created by Ghusse on 31/03/2017.
+ */
+public class DiffParser {
+  public DiffParser(){
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1186)